src/site/stages/build/process-cms-exports/transform.js

```diff
@@ -0,0 +1,55 @@
+const { mapKeys, camelCase } = require('lodash');
+const pageTransform = require('./transformers/page');
+
+const transformers = {
+  page: pageTransform,
+};
+
+const missingTransformers = new Set();
+
+/**
+ * Returns the proper function for transforming a specific entity type.
+ * If no transformer exists for the entity type, it returns a function
+ * that will return the entity unmodified, and it will log a warning.
+ *
+ * @param {String} entityType - The type of the entity
+ * @return {Function} - A function that accepts an entity and transforms it
+ */
+function getEntityTransformer(entityType) {
+  let entityTransformer = entity => entity;
+
+  if (entityType in transformers) {
```

Comment:
:thinking: Why use `in` instead of `if (transformers[entityType]) { ... }`?

Reply:
I don't have a particularly strong opinion on this, but I feel like using `in` makes things a bit more explicit.  `if(transformers[entityType])` could be checking if the property value is `false`, rather than `undefined`.